client/components/chart/index.js

```diff
@@ -399,6 +416,10 @@ Chart.propTypes = {
 	 * What type of data is to be displayed? Number, Average, String?
 	 */
 	valueType: PropTypes.string,
+	/**
+	 * Render a chart placeholder to signify an in-flight data request.
+	 */
+	isRequesting: PropTypes.string,
```

Comment:
Should this be a `boolean`? I'm getting a warning:

![screenshot 2018-11-22 09 55 58](https://user-images.githubusercontent.com/1160732/48888914-d98ac980-ee3c-11e8-9cea-6bba61876fae.png)